.swiftformat

```diff
@@ -36,3 +36,9 @@
 
 # the ordering doesn't match linter's
 --disable specifiers
+
+# conflicts with our Swiftlint explicit ACL rule
+--disable extensionAccessControl
+
+# conflicts with Swiftlint opening_brace rule
+--disable wrapMultilineStatementBraces
```

Comment:
SwiftFormat wanted us to start using a newline for opening braces following multiline conditionals, e.g.
```swift
if let a = x,
    let b = y {
    // ...
}
```

would become 
```swift
if let a = x,
    let b = y 
{
    ...
}
```

SwiftLint did not like this because it wants the opening brace to be on the same line as the end of the statement. I have like a 3/10 preference for the way we've always done it so stuck with that, but if you feel strongly the other way happy to switch

Reply:
I prefer the latter (more than 3/10 :smile: ) because otherwise it's kinda hard to parse the difference between the `if` portion of the statement and the actual body (e.g. here https://github.com/mongodb/mongo-swift-driver/blob/636cd541225754a8956e42419461a433bd23b113/Sources/MongoSwift/MongoError.swift#L354).

Ideally the various `let` keywords would be aligned too, but I'm not sure if that's an option.

Reply:
that's a good point. let's turn it on then 🙂